packages/terminal/src/browser/style/terminal-search.css

```diff
@@ -0,0 +1,112 @@
+/********************************************************************************
+ * Copyright (C) 2019 Red Hat, Inc. and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+.theia-search-terminal-widget-parent {
+    background: var(--theia-sideBar-background);
+    position: absolute;
+    margin: 0px;
+    border: var(--theia-border-width) solid transparent;
+    padding: 0px;
+    top: 1px;
+    right: 19px;
+
+    z-index: 10;
+}
+
+.theia-search-terminal-widget-parent .theia-search-elem-box {
+    display: flex;
+    margin: 0px;
+    border: var(--theia-border-width) solid transparent;
+    padding: 0px;
+    align-items: center;
+    color: var(--theia-input-foreground);
+    background: var(--theia-input-background);
+}
+
+.theia-search-terminal-widget-parent .theia-search-elem-box input {
+    margin-left: 5px;
+    padding: 0px;
+    width: 100px;
+    height: 18px;
+    color: inherit;
+    background-color: inherit;
+    border: var(--theia-border-width) solid transparent;
+    outline: none;
+}
+
+.theia-search-terminal-widget-parent .search-elem {
+    border: var(--theia-border-width) solid transparent;
+    background-position: center;
+    background-repeat: no-repeat;
+    height: 18px;
+    width: 18px;
+    margin: 1px;
+    opacity: 0.7;
+    outline: none;
+}
+
+.search-elem:hover {
```

Comment:
it has to be scoped by some `theia-` class, otherwise it is global class name and can break any other widgets making use of `search-elem` class as well